layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,42 @@
+<script src="/js/keps/kep_search.js"></script>
+<div >
+  <dir class="d-flex">
+    <div class="col">
+      <div class="input-group mt-2 col-md-12">
+        <input type="text" name="search" id="keps_search" class="form-control" placeholder="Search KEP(s) by keyword" />
+      </div>
+    </div>
+    <div class="col ml-auto">
+      <div class="input-group mx-auto mt-2 col-md-12" style="width: 400px;">
+        <label for="keps">Choose SIG group:</label>
+        <select class="custom-select" name="sig group" id="sig_group_select">
+          <option selected="selected" value="all">Choose...</option>
+          {{ range $data_sig := .Site.Data.sigs.sigs.sigs }}
```

Comment:
We don't need to pull in the complete sigs.yaml to just get the name of the SIG. The KEP data already has all the SIGs in each KEP entry. We can think about this filter by SIG feature in a later iteration.

Note: The list of SIGs can also be populated by JavaScript at page load by iterating over the values in SIG column.

Reply:
Yeah, I wanted to have options to choose complete list of SIGs we have. So lets say `sig-usability` keps we do not have in enhancements repo, but we still like to show it in dropdown.